src/scss/tom-select.bootstrap5.scss

```diff
@@ -142,14 +131,11 @@ $select-arrow-offset: calc(#{$select-padding-x} + 5px) !default;
 .#{$select-ns}-control {
 	@include box-shadow($input-box-shadow);
 	@include transition($input-transition);
+
 	display:flex;
 	align-items: center;
 
-	&.dropdown	-active {
```

Comment:
This was removed, because the SCSS as it was did nothing `&.dropdown	-active` is not valid, and looking at the html, I can't see any instance of  a `.dropdown-active` inside a `.ts-control`